docs/streams/developer-guide/memory-mgmt.html

```diff
@@ -168,7 +168,15 @@
       <h2><a class="toc-backref" href="#id3">RocksDB</a><a class="headerlink" href="#rocksdb" title="Permalink to this headline"></a></h2>
       <p> Each instance of RocksDB allocates off-heap memory for a block cache, index and filter blocks, and memtable (write buffer). Critical configs (for RocksDB version 4.1.0) include
         <code class="docutils literal"><span class="pre">block_cache_size</span></code>, <code class="docutils literal"><span class="pre">write_buffer_size</span></code> and <code class="docutils literal"><span class="pre">max_write_buffer_number</span></code>.  These can be specified through the
-        <code class="docutils literal"><span class="pre">rocksdb.config.setter</span></code> configuration.</li>
+        <code class="docutils literal"><span class="pre">rocksdb.config.setter</span></code> configuration.</p>
+      <p>We further recommend, to change RocksDB's default memory allocator, because the default allocator may lead to increased memory consumption.
+        To change the memory allocator to <code>jemalloc</code>, you need to set the an environment variable before you start your Kafka Streams application:</p>
```

Comment:
```suggestion
        To change the memory allocator to <code>jemalloc</code>, you need to set the environment variable before you start your Kafka Streams application:</p>
```

Reply:
I guess it should be `set an environment variable` as we don't include the name?

Or should we change it to `set the environment variable LD_PRELOAD`?

Reply:
Mm good point. I think `set the environment variable LD_PRELOAD` is best